File: 351004/Narivonchik/rest1/rest1/src/core/entities/Note.cs
namespace rest1.core.entities;

public class Note : Entity
{
    public long NewsId { get; set; }

    public string Content { get; set; } = string.Empty;

    public string Country { get; set; } = string.Empty;
    
    // lookup object
    public News? News { get; set; }
   
}

File: 351001/Boukhvalova/module-discussion/src/main/java/by/boukhvalova/distcomp/entities/Note.java
package by.boukhvalova.distcomp.entities;

import jakarta.validation.constraints.Size;
import lombok.*;
import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table("tbl_note")
public class Note {

    @PrimaryKey
    private Long id;

    @Size(min = 2, max = 2048)
    String content;

    @Column("tweet_id")
    private long tweetId;

    @Column("user_id")
    private long userId;

    private String country;
}
